ledger/streaming/client/src/http_fetcher.rs

```diff
@@ -1,16 +1,18 @@
 // Copyright (c) 2018-2022 The MobileCoin Foundation
 
-//! A [Fetcher] that downloads [BlockData] from the given URI.
+//! A [Fetcher] that downloads [ArchiveBlocks] from the given URI.
```

Comment:
Since this was updated should the doc comment on the `HttpBlockFetcher` be updated as well?

Also is `URI` correct here? It looks like `URL` is used in `new()` and `from_url()`